workspaces/liferay-learn-workspace/client-extensions/liferay-learn-global-css/assets/global.css

```diff
@@ -1737,6 +1737,51 @@ pre[class*='language-'] {
 	font-weight: 400;
 }
 
+.voting-box__content {
```

Comment:
The global.css contains many classes and lines of code.
If if you don't scope it, it will be hard to maintain. 
Please, scope the classes. 

```
.learn-recipe__main-content {
   .voting-box... 
}
```